ddtrace/profiling/collector/_memalloc_heap_map.cpp

```diff
@@ -77,35 +77,49 @@ CWISS_DECLARE_FLAT_MAP_POLICY(HeapSamples_policy32, void*, traceback_t*, (key_ha
 CWISS_DECLARE_HASHMAP_WITH(HeapSamples, void*, traceback_t*, HeapSamples_policy32);
 #endif
 
-typedef struct memalloc_heap_map_t
+// Pimpl implementation
```

Comment:
Not sure this `pimpl` is needed.  The advantage is that it hides the cwisstable implementation, the downside it an additional layer of indirection

Reply:
removed